src/components/Header/__tests__/__snapshots__/header.test.tsx.snap

```diff
@@ -45,29 +45,17 @@ exports[`Tests for Header component renders correctly 1`] = `
           </a>
         </li>
         <li
-          className="nav__tabs only-desktop"
+          className="nav__tabs"
         >
           <a
+            activeClassName="active"
             className="activeStyleTab"
-            href="https://nodejs.org/en/docs/"
-            rel="noopener noreferrer"
-            target="_blank"
+            href="/docs"
```

Comment:
Can you change back to the current docs? https://nodejs.org/en/docs  The new docs still have a lot of work to do so we shouldn't make them public yet.
Other then that LGTM 👍